Account/CreateAccount/Program.cs

```diff
@@ -1,14 +1,13 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-using Common_Utils;
-using System.Threading.Tasks;
-using Azure.Identity;
 using Azure;
 using Azure.Core;
+using Azure.Identity;
 using Azure.ResourceManager;
-using Azure.ResourceManager.Resources;
 using Azure.ResourceManager.Media;
 using Azure.ResourceManager.Media.Models;
+using Azure.ResourceManager.Resources;
+using Common_Utils;
```

Comment:
Why is `Common_Utils` needed? (where possible, I'm trying to make the samples standalone so users don't need to look in multiple files to understand how to use the SDK).

Reply:
I did not change the code, just did a reordering.
Common_Utils is used for loading the settings.

// Load configuration using the Common Utilities library
ConfigWrapper config = Common_Utils.DotEnv.LoadEnvOrAppSettings();

We need to define our policy for the settings for all the samples.
We can get rid of common_Utils but it means will will duplicate the ConfigWarapper class and LoadEnvOrAppSettings function in each sample.

Reply:
Fine for now, but agree we should try to make them more standalone.  Reason is that @IngridAtMicrosoft  is trying to get these to show up as a single page in the docs with all the code needed in one file, like the TypeScript and Python samples.  Less complex spaghetti code is good for the docs.